content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Open Source Hub"
```

Comment:
I think you should replace "Open Source Hub" with "Ask the Experts & Demos" to mirror the posts I had written before the event.

I kept repeating "Open Source Hub" in my posts, but I'm not sure anybody but us identified it as such during the event :-P